best-practices/tech_lead.md

```diff
@@ -0,0 +1,318 @@
+  - (this is one of the reasons why the TL should not take blocking tickets or large tickets -- the TL priority is to keep the other devs unblocked)
+- If you think a dev took a ticket but might unknowingly be missing some context or understanding, reach out.
+### With PO or Designer
+- Exchange info with PO or Designer - ask questions to clarify work in tickets, or relative priorities, or whatever.  
+
+## Coding Goals (i.e. Maint / Tech Debt)
+- “You're constantly trading off short-term delivering things versus long-term tech debt or investing in platforms of things for the future.” - https://blog.coleadership.com/what-does-it-take-to-be-a-successful-tech-lead/
+- “If we only focus on today, we are going to bury ourselves in tech debt or in systems that aren't flexible enough to handle future use cases.” - https://blog.coleadership.com/what-does-it-take-to-be-a-successful-tech-lead/
+
+### Leave the code better than it was when work-cycle began
+Balance refactoring, more/better testing and introducing new technologies against desired feature development.
+- adherence to best practices
+- replace a technology?
+- move towards a cleaner/better/more contemporary code structure?  
+  - best practices change over time:  OO, rails, rspec
+- improve testing?
+  - methodology, coverage, ...
+- improve maintainability
+  - sometimes simple things like a rename or simple refactor make the code more easily understood
+
+## Tickets
+(aka github issues)
+
+- Prepare github issues before the work cycle starts, so devs can get started being happy coders.
+- Create / edit / comment on issues as new information comes to light
+- Ensure open issues intended for devs are actionable by devs
+  - are there design or PO decisions that are needed for the work in the ticket to be completed?
+    - it is the responsibility of the TL to ensure the necessary information is added to the ticket.
+- It's okay to add comments to the ticket like "talk to me before you start coding for this" or "this will require some familiarity with xxx"
+- Identify when sign off is needed by the PO or the designer and indicate that in the ticket.
+
+## Board Grooming
+This is a big part of being tech lead.  The board is our way of knowing the state of work, our way to determine devs who might benefit from help without realizing they need it (e.g. "Naomi took that ticket but I need to make sure she knows about XXX"), our way to ensure there is always prioritized work available when a developer is looking for a task.
+
+The board can also be a tool for the PO or a Manager:  can they tell what is in progress, what is complete, how issues relate to the Epics (more below) for the work cycle, etc.?
+
+Q: _How often should the TL look at / groom the board?_
+
+A: As often as possible throughout the day. Absolute minimum at 2-3 times / day.  
+
+Q: _What does grooming the board mean?_
+- Is the ready column populated with prioritized, actionable work? (more below)
```

Comment:
I think you may be missing an `A: ` hereish

Reply:
yeppers.  addressed.